packages/Button/src/view/ButtonView.tsx

```diff
@@ -2,26 +2,38 @@ import React from 'react';
 import { buildClassNames } from '@blaze-react/utils';
 import { IButtonViewProps } from '../types';
 
-export const ButtonView = ({ children, classes = '', ...rest }: IButtonViewProps): JSX.Element => {
+export const ButtonView = ({ children, classes = '', label, ...rest }: IButtonViewProps): JSX.Element => {
   const { modifiers = [], disabled, type = 'button' } = rest;
 
   const DeprecatedButton = () => {
     console.warn('Modifiers will be deprecated in the near future. You should use CSS classes classes instead.');
 
-    const formatedModifiers: string = modifiers.map((modifier) => `button--${modifier}`).join(' ');
+    const formattedModifiers: string = modifiers.map((modifier) => `button--${modifier}`).join(' ');
     const buttonClassNames: string = buildClassNames('button', {
-      [formatedModifiers]: !!modifiers,
+      [formattedModifiers]: !!modifiers,
     });
 
     return (
-      <button disabled={disabled} className={buttonClassNames} type={type} {...rest}>
+      <button
+        disabled={disabled}
+        className={buttonClassNames}
+        type={type}
+        aria-label={label}
+        {...rest}
+      >
         {children}
       </button>
     );
   };
 
   const BlazeButton = (
-    <button type="button" className={classes} {...rest} disabled={disabled}>
+    <button
```

Comment:
Can you add a test with a snapshot to show the label prop being used as the aria-label

Reply:
added!